src/main/config/cromwell.conf.ctmpl

```diff
@@ -0,0 +1,57 @@
+{{with $environment := env "ENVIRONMENT"}}{{$keyname := printf "secret/dsde/%s/cromwell/cromwell.conf" $environment}}{{with vault $keyname}}
+
+include "application.conf"
+
+backend {
+  backend = "JES"
+
+  jes {
+    applicationName = "cromwell"
+    project = "{{.Data.jes_project}}"
+    baseExecutionBucket = "{{.Data.jes_base_execution_bucket}}"
+    endpointUrl = "{{.Data.jes_endpoint_url}}"
+    dockerhubCredentialsPath = "{{.Data.jes_dockerhub_credentials_path}}"
+    authenticationMode = "{{.Data.jes_authentication_mode}}"
+    maximumPollingInterval = 600
```

Comment:
it's configurable... it's just not something that should be in vault as a 'secret'.  is that what you're asking?